catalogue/webapp/components/ExpandedImage/ExpandedImage.js

```diff
@@ -30,26 +32,32 @@ const Box = styled.div`
   background-color: ${({ theme }) => theme.colors.pumice};
   display: flex;
   flex-direction: column;
-  margin-bottom: 20px;
-  padding: 20px;
-  
+
   ${({ index, theme }) => theme.media.small`
     width: calc(200% + ${theme.gutter.small}px);
+    padding:  ${theme.gutter.small}px;
+    margin-bottom: ${theme.gutter.small}px;
```

Comment:
In an attempt to keep spacing as systematic as possible, we (try to) use a `<Space />` styled component anywhere we're adding space. It takes one of five `size` values (`xs`, `s`, `m`, `l`, and `xl`), and these output different amounts of space per breakpoint.

It takes `v` and `h` objects to handle vertical and horizontal spacing.

e.g.
```
<Space
  h={{ size: 's', properties: ['padding-left', 'padding-right'] }}
  v={{ size: 's', properties: ['padding-top', 'padding-bottom', 'margin-bottom'] }}>
</Space>
```

It's Flow-typed, so the `h` and `v` keys can only have horizontal-ish and vertical-ish properties respectively.

By default it outputs a div, but you can use it as the basis for a different component, e.g.

```
const ShinyComponent = styled(Space).attrs({
  h: { size: 's', properties: ['padding-left', 'padding-right'] },
  v: {size: 's', properties: ['padding-top', 'padding-bottom', 'margin-bottom']}
})`

`
```
…or you could add the `h` and `v` props as attributes to `ShinyComponent` where it's used, if the values can differ from case to case.

[This file on Zeplin](https://app.zeplin.io/project/5aab926b4b6efde01259e959/screen/5cc189c2333a2a2cd7dbc1fb) might make it a bit less abstract, but give us a shout if anything remains unclear.

Reply:
Got it, I'll switch to using `Space` where appropriate :+1: